playbooks/prometheus_agent.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 - name: Deploy Prometheus agent
-  hosts: "{{ prometheus_destination | default('controller-0') }}"
+  hosts: "{{ prometheus_destination | default('allovercloud') }}"
```

Comment:
Sorry, I think this was right the first time. I somehow had a picture in my head of an agent on each node that scrapes the local containers, but that is really not what we discussed. My apologies for the confusion.

 https://github.com/infrawatch/python-observabilityclient/pull/1#issuecomment-1184624887

Reply:
```suggestion
  hosts: "{{ prometheus_destination | default('controller-0') }}"
```